lib/krb5/rd_req.c

```diff
@@ -220,7 +221,20 @@ krb5_decrypt_ticket(krb5_context context,
 	    return KRB5KRB_AP_ERR_TKT_EXPIRED;
 	}
 
-	if(!t.flags.transited_policy_checked) {
+	if(t.flags.transited_policy_checked) {
+	    skip_transit_check = TRUE;
+	} else if(flags & KRB5_VERIFY_AP_REQ_SKIP_TRANSITED_CHECK) {
+	    skip_transit_check = TRUE;
+	} else {
+	    skip_transit_check = krb5_config_get_bool_default(context,
+							      NULL,
+							      FALSE,
+							      "libdefaults",
+							      "acceptor_skip_transit_check",
+							       NULL);
```

Comment:
extra space here

Reply:
acceptor is a GSS-ism so, probably this can be renamed to something that uses Kerberos terminology given it lives in libkrb5.

Reply:
@lhoward first thanks very much for the review!

As I'm aiming to have things upstreamed in Heimdal and MIT,
lets discuss the option name in https://github.com/krb5/krb5/pull/1005
there I'm waiting for comments from @greghudson